packages/flame/test/components/advanced_button_component_test.dart

```diff
@@ -0,0 +1,204 @@
+import 'package:flame/components.dart';
+import 'package:flame/game.dart';
+import 'package:flame/src/events/flame_game_mixins/multi_tap_dispatcher.dart';
+import 'package:flame_test/flame_test.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  group('AdvancedButtonComponent', () {
```

Comment:
It would be good to add a golden test that ensures that the label components are centered too.

Reply:
The last time I tried them on a Mac, I had problems. My result did not match yours :))

Reply:
It's because the actions are running on Linux machines, if you create the test I can generate the goldens for you

Reply:
I'm embarrassed , but I never wrote them ((

Reply:
It's not hard, you can look at one of the current golden tests that we have for inspiration, for example this one:
https://github.com/flame-engine/flame/blob/main/packages/flame/test/camera/camera_component_test.dart#L311-L346

Reply:
It's not hard, you can look at one of the current golden tests that we have for inspiration, for example this one:
https://github.com/flame-engine/flame/blob/main/packages/flame/test/camera/camera_component_test.dart#L311-L346